Desktop/data/importers/excel/excelimportcolumn.cpp

```diff
@@ -0,0 +1,32 @@
+#include "excelimportcolumn.h"
+#include "timers.h"
+
+ExcelImportColumn::ExcelImportColumn(ImportDataSet* importDataSet, std::string name) : ImportColumn(importDataSet, name)
+{
+}
+
+ExcelImportColumn::ExcelImportColumn(ImportDataSet *importDataSet, std::string name, long reserve) : ImportColumn(importDataSet, name)
+{
+	_data.reserve(reserve);
+}
+
+ExcelImportColumn::~ExcelImportColumn()
+{
+	JASPTIMER_SCOPE(ExcelImportColumn::~ExcelImportColumn());
+	_data.clear();
```

Comment:
This not really necessary, as the vector will get cleaned up in any case by the destructor.